### PR TITLE
"struct_lookup" action for bureaucracy

### PR DESCRIPTION
**Pull request by solewniczak**
Action allows create struct lookup entries using bureaucracy form. Syntax:
  <form>
  action struct_lookup
  ...
  </form>

<form> can contain any number of struct_field or struct_schema. Data will be saved to respective lookup schemas.

### CODE REVIEW

**Comment by solewniczak** [comment]
Works well with  #356

File: helper/lookup.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+            if(!isset($tosave[$tbl])) $tosave[$tbl] = array();
+            $tosave[$tbl][$lbl] = $field->getParam('value');
+        }
+
+        foreach($tosave as $table => $data) {
+            $access = AccessTable::byTableName($table, 0, 0);
+            if (!$access instanceof AccessTableLookup) continue;
+
+            if(!$access->getSchema()->isEditable()) {
+                throw new StructException('lookup save error: no permission for schema');
```
Comment (splitbrain) [inline helper/lookup.php]: Does throwing a Struct Exception in the bureaucracy context work? I don't think bureaucracy is catching exceptions. Instead a msg() should be shown and false be returned.

**Comment by splitbrain** [comment]
can you update the docs again? thanks!

**Comment by solewniczak** [comment]
Ok. No problem.

**Pull request merged**